packages/account-v2/src/constants/routes.ts

```diff
@@ -1,22 +1,22 @@
-export const P2P_ROUTE = '/cashier/p2p';
+export const p2pRoute = '/cashier/p2p';
 
-export const ACCOUNT_V2_DEFAULT_ROUTE = '/account-v2';
+export const accountV2DefaultRoute = '/account-v2';
 
-export const ACCOUNT_V2_ROUTES = {
-    AccountLimits: `${ACCOUNT_V2_DEFAULT_ROUTE}/account_limits`,
-    ApiToken: `${ACCOUNT_V2_DEFAULT_ROUTE}/api_token`,
-    CloseAccount: `${ACCOUNT_V2_DEFAULT_ROUTE}/close_your_account`,
-    ConnectedApps: `${ACCOUNT_V2_DEFAULT_ROUTE}/connected_apps`,
-    EmailAndPassword: `${ACCOUNT_V2_DEFAULT_ROUTE}/email_and_passwords`,
-    FinancialAssessment: `${ACCOUNT_V2_DEFAULT_ROUTE}/financial_assessment`,
-    Languages: `${ACCOUNT_V2_DEFAULT_ROUTE}/languages`,
-    LoginHistory: `${ACCOUNT_V2_DEFAULT_ROUTE}/login_history`,
-    PersonalDetails: `${ACCOUNT_V2_DEFAULT_ROUTE}/personal_details`,
-    ProofOfAddress: `${ACCOUNT_V2_DEFAULT_ROUTE}/proof_of_address`,
-    ProofOfIdentity: `${ACCOUNT_V2_DEFAULT_ROUTE}/proof_of_identity`,
-    ProofOfIncome: `${ACCOUNT_V2_DEFAULT_ROUTE}/proof_of_income`,
-    ProofOfOwnership: `${ACCOUNT_V2_DEFAULT_ROUTE}/proof_of_ownership`,
-    SelfExclusion: `${ACCOUNT_V2_DEFAULT_ROUTE}/self_exclusion`,
-    TradingAssessment: `${ACCOUNT_V2_DEFAULT_ROUTE}/trading_assessment`,
-    TwoFactorAuthentication: `${ACCOUNT_V2_DEFAULT_ROUTE}/two_factor_authentication`,
+export const accountV2Routes = {
```

Comment:
I think we should follow airbnb guides on constants
https://github.com/airbnb/javascript?tab=readme-ov-file#naming--uppercase